docs/guided-tour/getting-started.md

```diff
@@ -62,4 +221,4 @@ customizing it for your use case.
 
 If you want to find out more about the product and participate in a live Q&A
```

Comment:
I am not sure what will be done re. Semaphore Class but I'll check with Darko. It is possible that we will need to reword this part. Everything else is 👌

Reply:
Remove this paragraph. The Class is on hold for a while now.